src/transformers/models/whisper/processing_whisper.py

```diff
@@ -76,11 +76,9 @@ def get_decoder_prompt_ids(self, task=None, language=None, no_timestamps=True):
 
     def __call__(self, *args, **kwargs):
         """
-        When used in normal mode, this method forwards all its arguments to WhisperFeatureExtractor's
-        [`~WhisperFeatureExtractor.__call__`] and returns its output. If used in the context
-        [`~WhisperProcessor.as_target_processor`] this method forwards all its arguments to WhisperTokenizer's
-        [`~WhisperTokenizer.__call__`]. Please refer to the doctsring of the above two methods for more information.
-
+        Forwards the `audio` argument to WhisperFeatureExtractor's [`~WhisperFeatureExtractor.__call__`] and the `text`
```

Comment:
Super nice! It looks you are the first one who clean this up! We have other processor to do (for other PR).